v2/test/datadog.spec.ts

```diff
@@ -0,0 +1,238 @@
+import { App, Stack } from "aws-cdk-lib";
```

Comment:
All tests were modified to use V2 imports, and V2 testing functions such as:

`Template.fromStack(stack).hasResourceProperties("AWS::Lambda::Function", {`
vs.
`expect(stack).toHaveResourceLike("AWS::Lambda::Function", {` in v1

Same is true for ALL test files